docs/pages/modules/config-plugin-and-native-module-tutorial.mdx

```diff
@@ -307,25 +334,61 @@ class ExpoNativeConfigurationModule() : Module() {
 }
 ```
 
-## 6. Running your module
+On iOS, you can read the content of an **Info.plist** property using the `Bundle.main.object(forInfoDictionaryKey: "")` method. To access the `"MY_CUSTOM_API_KEY"` value added earlier, update the **ios/ExpoNativeConfigurationModule.swift** file as shown:
 
-With our native modules reading the fields we added to the native files, we can now run the example app and access our custom API key through the `ExamplePlugin.getApiKey()` function.
+```swift ios/ExpoNativeConfigurationModule.swift
+import ExpoModulesCore
+
+public class ExpoNativeConfigurationModule: Module {
+  public func definition() -> ModuleDefinition {
+    Name("ExpoNativeConfiguration")
+
+    Function("getApiKey") {
+     return Bundle.main.object(forInfoDictionaryKey: "MY_CUSTOM_API_KEY") as? String
+    }
+  }
+}
+```
+
+</Step>
+
+<Step label="6">
+
+## Running your module
+
+With your native modules reading the fields added to the native files, you can now run the example app and access your custom API key using the `ExamplePlugin.getApiKey()` function.
 
 <Terminal
   cmdCopy="cd example && npx expo run:ios"
   cmd={[
     '$ cd example',
     '# execute our plugin and update native files',
     '$ npx expo prebuild',
-    '# Run the example app on iOS',
-    '$ npx expo run:ios',
     '# Run the example app on Android',
     '$ npx expo run:android',
+    '# Run the example app on iOS',
+    '$ npx expo run:ios',
   ]}
 />
 
+</Step>
+
 ## Next steps
 
-Congratulations, you have created a simple yet non-trivial config plugin that interacts with an Expo module for Android and iOS!
+Congratulations, you have created a config plugin that interacts with an Expo module for Android and iOS!
+
+If you want to challenge yourself and make the plugin more versatile, this exercise is open for you. Modify the plugin to allow any arbitrary set of config keys and values to be passed in, and add functionality to read arbitrary keys from the module.
 
-If you want to challenge yourself and make the plugin more versatile we leave this exercise open to you. Try modifying the plugin to allow for any arbitrary set of config keys/values to be passed in and adding the functionality to allow for the reading of arbitrary keys from the module.
+<BoxLink
+  title="Expo Modules API Reference"
+  description="Create native modules using Kotlin and Swift."
```

Comment:
```suggestion
  description="A reference to create native modules using Kotlin and Swift."
```

The description wasn't self explanatory so let's add the context about the it being a reference.